ecdsa.go

```diff
@@ -188,27 +189,26 @@ func hashBytes(data []byte) []byte {
 	return buf[:]
 }
 
-// Verify verifies the signature in r, s of hash using the public key, pub. Its
-// return value records whether the signature is valid.
-func Verify(pub *ecdsa.PublicKey, hash, data []byte) error {
-	if r, s := unmarshalXY(hash); r == nil || s == nil {
-		// panic("could not unmarshal r / s")
+// Verify verifies the signature of msg using the public key pub. It returns
+// nil only if signature is valid.
+func Verify(pub *ecdsa.PublicKey, sig, msg []byte) error {
```

Comment:
I think it is better also to also change order of `msg` and `sig`, because go's stdlib and many other libraries (e.g. kyber) accept first message then signature.
https://golang.org/pkg/crypto/ed25519/#Verify
https://golang.org/pkg/crypto/ecdsa/#Verify
https://golang.org/pkg/crypto/rsa/#VerifyPKCS1v15

Reply:
I thought about it, but decided to make small changes at first. 

Let's see @im-kulikov review. If the general idea of this PR is okay, I don't mind to change the order of parameters for better consistency.

Reply:
@fyrchik I will do it soon, thx (see https://github.com/nspcc-dev/neofs-crypto/issues/5)